sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -368,8 +368,11 @@ def get_queue_receiver(
          performance but increase the chance that messages will expire while they are cached if they're not
          processed fast enough.
          The default value is 0, meaning messages will be received from the service and processed one at a time.
-         In the case of prefetch_count being 0, `ServiceBusReceiver.receive` would try to cache `max_message_count`
-         (if provided) within its request to the service.
+         In the case of prefetch_count being 0, `ServiceBusReceiver.receive_messages` would try to cache
+         `max_message_count` (if provided) within its request to the service.
+         **WARNING: If prefetch_count > 0 and RECEIVE_AND_DELETE mode is used, all prefetched messages will stay in
+         the in-memory prefetch buffer until they're received into the application. If the application ends before
+         the messages are received into the application, those messages are irrecoverable (lost).
```

Comment:
If we have to clarify the point by adding `"(lost)"` - maybe we should rephrase :)
Perhaps something like `"those messages will be lost and unable to be recovered"`